suite flag "m" conflicts with application flag

Dear
Our application already defines a flag using the "m" value.
When running a main_test.go using testify, run panics with:
panic: /tmp/gb457819402/xxx.test flag redefined: m

Indeed, in suite.go, see:
var matchMethod = flag.String("m", "", "regular expression to select tests of the suite to run")

Would there be any way to avoid that ?
Cheers
WT

--- a/suite/suite.go
+++ b/suite/suite.go
@@ -12,7 +12,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-var matchMethod = flag.String("m", "", "regular expression to select tests of the suite to run")
+var matchMethod = flag.String("testify.m", "", "regular expression to select tests of the testify suite to run")
 
 // Suite is a basic testing suite with methods for storing and
 // retrieving the current *testing.T context.

